### ISSUE TITLE
NVDA/Narrator does not read the line number. Line numbers are not programmatically linked to the corresponding code content: A11y_Azure Databases extension for VS Code_Info and relationship.

### ISSUE THREAD

**Issue by PadminiGariki** [microsoft/vscode-cosmosdb#2928]
### GitHub Tags:
#A11ySev2; #A11yTCS; #DesktopApp; #BM_Azure Databases extension for VS Code_Win32_Nov2025; #Win32; #Win11; #FTP; #A11yMAS; #Azure Databases extension for VS Code; #Benchmark; #AILimited; #MAS1.3.1; #Info and Relationship; #Element:text; #Screen Reader; #NVDA; #Narrator; #STP;

"Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility tester after bug verification."

"[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier."

### Environment Details:
Application Name: Azure Databases extension for VS Code
OS Version: Windows 11 enterprise 24H2. (OS build 26120.3360)
Screen Reader: Narrator, NVDA (2025.3.2)

### Steps to Reproduce

1. Turn on NVDA/Narrator.
2. Open VS Code
3. Open the "Azure" Sidebar
4. Sign In to your Azure Account (skip if already signed in)
5. Expand the "Cosmosdb-Portalteam-generaltest-msft" subscription node
6. Expand the "Azure Cosmos DB" node
7. Expand the "a11y-nosql" Cosmos DB account
8. Expand the "SampleDB" database node
9. Expand the "SampleContainer" node.
10. Tab till "Create query" and press enter.
11. Run a simple query such as "SELECT TOP 1000 * FROM c"
12. Tab till "Results" tab and press enter.
13. Tab till "Add New Line" and press enter. 
14. Observed the issue NVDA/Narrator does not read the line number. Line numbers are not programmatically linked to the corresponding code content.

### Actual Results:
NVDA/Narrator does not read the line number. Line numbers are not programmatically linked to the corresponding code content.

### Expected Results:
Each code line should be programmatically associated with its line number so the NVDA/Narrator can announce it correctly and there can easily edit it.

### User Impact:
Screen reader users cannot understand which line they are editing or reviewing. This causes confusion and makes navigation and debugging extremely difficult. Lack of proper association reduces accessibility compliance for code editors.

### Attachment:

[2928_A11y_Azure Databases extension for VS Code_Narrator.zip](https://github.com/user-attachments/files/25068915/2928_A11y_Azure.Databases.extension.for.VS.Code_Narrator.zip)

[2928_A11y_Azure Databases extension for VS Code_NVDA.zip](https://github.com/user-attachments/files/25068919/2928_A11y_Azure.Databases.extension.for.VS.Code_NVDA.zip)

<img width="1365" height="711" alt="Image" src="https://github.com/user-attachments/assets/2e405c51-6ae6-436d-ab68-4deeb2bc0486" />
